### PR TITLE
Add README.md file

### PR DESCRIPTION
**Pull request by dersim-davaod**
Checks:

* [ ] Update changelog
* [x] Rebase to `dev` branch
* [x] Assign reviewers

Linked issues:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+- Scene segmentation;
+- Sign detection;
+- Safety alerts;
+- Object detection;
+- Lane detection.
+
+Vision SDK for iOS contains several modules which built on top of Vision Core library (the implementation of Vision Core is in private repository):
+  1. `Vision SDK` - an end-user SDK with a lot of helpers such as:
+      * Device camera session implementation;
+      * Interfaces and additional functions for an API data structures;
```
Comment (chezzdev) [inline README.md]: Not quite follow what it means.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      * Views of segmentation, detection, for debugging purposes.
```
Comment (chezzdev) [inline README.md]:
```suggestion
      * View components for visualization of segmentation, detection and AR routing.
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  1. `VisionAR SDK` for AR experience on iOS platform. It has own API to control:
```
Comment (chezzdev) [inline README.md]:
```suggestion
  1. `MabpoxVisionAR` for AR experience on iOS platform. It has own API to control:
```

File: README.md [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+      * speed limits;
+      * collision alerts;
+      * etc.
+
+# Getting Started
+
+## Requirements
+
+The Vision SDK for iOS is written in Swift 4.2 and can be used with:
+  - iOS 11.2 and higher (iOS 13 support is under development);
```
Comment (chezzdev) [inline README.md]:
```suggestion
  - iOS 11.2 and higher;
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+
+# Versioning
+
+We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on repository](https://github.com/mapbox/mapbox-vision-ios/tags).
+
+Changes in repository is in sync with changes in Vision Core library (which is under-the-hood of Vision SDK).
+
+# License
+
+For details, read our [terms of service](https://www.mapbox.com/tos/#vision) and [privacy policy](https://www.mapbox.com/privacy/).
```
Comment (chezzdev) [inline README.md]: Please link to license file instead.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+
+## Requirements
+
+The Vision SDK for iOS is written in Swift 4.2 and can be used with:
+  - iOS 11.2 and higher (iOS 13 support is under development);
+  - iPhone 6s or newer.
+  
+You can find all requirements at [Documentation page](https://docs.mapbox.com/ios/vision/overview/#requirements).
+
+## Build process
```
Comment (chezzdev) [inline README.md]: It's more of a installation instructions rather than build process

File: README.md [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+
+To set up the Vision SDK you will need to download the SDK, install the frameworks relevant to your project, and complete a few configuration steps. You can find all details at [Documentation page](https://docs.mapbox.com/ios/vision/overview/#getting-started).
+
+# Documentation
+
+The lastest version of documentation is available at [Vision's page](https://docs.mapbox.com/ios/vision).
+
+# Contributing
+
+Please see our contribution guide for more details about the process.
```
Comment (chezzdev) [inline README.md]: Do we need to mention it while we don't have formulated guide?

File: README.md (hunk 6 above)
Comment (dersim-davaod) [reply README.md]: Agree.

File: README.md (hunk 7 above)
Comment (dersim-davaod) [reply README.md]: Removed for now.

File: README.md (hunk 1 above)
Comment (dersim-davaod) [reply README.md]: Me too :)
It was copied from Vision Architecture document. I suggest to remove that line.

**Comment by dersim-davaod** [comment]
> Please revise the section with modules description
@chezzdev I've removed the description of modules.

**Pull request merged**